sys/phydat/phydat_str.c

```diff
@@ -106,14 +106,14 @@ const char *phydat_unit_to_str(uint8_t unit)
         case UNIT_M:        return "m";
         case UNIT_M2:       return "m^2";
         case UNIT_M3:       return "m^3";
-        case UNIT_G:        return "g";
+        case UNIT_G_FORCE:  return "gₙ";
         case UNIT_DPS:      return "dps";
-        case UNIT_GR:       return "G";
+        case UNIT_GRAM:     return "g";
         case UNIT_A:        return "A";
         case UNIT_V:        return "V";
         case UNIT_W:        return "W";
         case UNIT_DBM:      return "dBm";
-        case UNIT_GS:       return "Gs";
+        case UNIT_GAUSS:    return "G";
```

Comment:
Why would you go from less ambiguous to more?

```suggestion
        case UNIT_GAUSS:    return "Gs";
```

Reply:
`G` is the correct symbol for Gauss.

Reply:
While both are correct, the actual SI unit for this is Tesla

Reply:
So deprecate `UNIT_GAUSS` right away?

Reply:
No, and the conversion factor is 10000 anyway.
This PR started by fixing a *wrong* and making some units less ambiguous, it shouldn’t cause petty disruption.